inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -203,7 +242,7 @@ class Coil:
 
     Attributes: 
         curve_coefficients (np.ndarray): Array with Fourier coefficients 
-        defining the coil.
+          defining the coil.
```

Comment:
I am assuming is a 2D array (real + imaginary part). Do you want to be more explicit about this? Also, this has a size to? Has it been defined before (e.g. in the constructor) at some point so you can "check" that all is correct?

Reply:
I believe that it is fairly well described in the class Docstring just above that. There I have the whole explanation about the Fourier Series.